src/introduction/qemu.md

```diff
@@ -0,0 +1,33 @@
+# Running QEMU
+
+This section is optional; some users of this guide will have been provided
+environments for building and running CHERI systems and programs.  If you have
+not, or if you also wish to be able to run things locally or in emulation,
+building and using our CHERI-RISC-V QEMU emulator is straightforward.
+
+First, follow the instructions in [Cross compilation](cross-compilation.md) to
+obtain `cheribuild`.  Then, it should suffice to run
+```
+~/cheri/cheribuild/cheribuild.py run-riscv64-purecap -d
+```
+After quite a lot of activity and some intermixed prompts about cloning
+dependencies, you should find yourself running a CheriBSD system.  Log in as
+`root` with no password.
+
+In general, re-running that command will fetch any updates and do incremental
+rebuilds of everything.  Leave off the `-d` if you just want to re-run the
+existing build, and note that rebuilds rebuild the entire virtual disk image,
```

Comment:
It's a bit weird to talk about dropping -d and then go back to rebuilds. Someone not paying attention might think that applies to `cheribuild run-riscv64-purecap` too.